ejb-remote/pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.jboss.quickstarts.eap</groupId>
-    <artifactId>jboss-ejb-remote-parent</artifactId>
```

Comment:
@rafabene : qstools reported this as a problem because the artifactId doesn't match the folder name.